draft-ietf-mls-architecture.md

```diff
@@ -1137,7 +1140,7 @@ future. Thus, MLS offers a version negotiation mechanism; this mechanism
 prevents version downgrade attacks where an attacker would actively rewrite
 messages with a lower protocol version than the messages originally offered by the
 endpoints. When multiple versions of MLS are available, the negotiation protocol
-guarantees that the version agreed upon will be the highest version supported in
+guarantees that the version agreed upon will be the most preferred version supported in
```

Comment:
"Most preferred" seems to raise the question of "preferred by whom".  (The answer is "the creator", but might not want to get into that here.)  Maybe just "best version"?

Reply:
I don't think it guarantees that they will get the "best" version. For instance, there might be disagreement about best.  How about
"The negotiation protocol guarantees that the creator is able to select the best version out of..."

Reply:
That works for me.